rdwatch/core/utils/worldview_nitf/raster_tile.py

```diff
@@ -61,25 +90,87 @@ def get_worldview_nitf_bbox(
     with rasterio.Env(
         GDAL_DISABLE_READDIR_ON_OPEN='EMPTY_DIR',
         GDAL_HTTP_MERGE_CONSECUTIVE_RANGES='YES',
-        GDAL_CACHEMAX=200,
-        CPL_VSIL_CURL_CACHE_SIZE=20000000,
-        GDAL_BAND_BLOCK_CACHE='HASHSET',
         GDAL_HTTP_MULTIPLEX='YES',
         GDAL_HTTP_VERSION=2,
-        VSI_CACHE='TRUE',
-        VSI_CACHE_SIZE=5000000,
+        CPL_VSIL_CURL_CHUNK_SIZE=524288,
     ):
-        with _download_nitf_image(capture.uri) as nitf_filepath:
-            with Reader(input=nitf_filepath) as img:
-                rgb = img.part(bbox)
-
-        if scale == 'default':
-            rgb.rescale(in_range=((0, 255),))
-        elif scale == 'bits':
-            if capture.bits_per_pixel != 8:
-                max_bits = 2**capture.bits_per_pixel - 1
-                rgb.rescale(in_range=((0, max_bits),))
-        elif isinstance(scale, list) and len(scale) == 2:  # scale is an integeter range
-            rgb.rescale(in_range=((scale[0], scale[1]),))
-
-        return rgb.render(img_format=format)
+        logger.info(f'Downloading WorldView NITF bbox: {bbox} scale: {scale}')
+        if capture.panuri is not None:
+            with Reader(input=capture.panuri) as pan_img:
+                logger.info(f'Downloading PanURI Chip: {capture.panuri}')
+                try:
+                    pan_chip = pan_img.part(bbox=bbox, dst_crs='epsg:4326')
+                except ValueError as e:
+                    logger.info(f'Value Error: {e} - {capture.panuri}')
+                    return None
```

Comment:
What circumstances would cause a `ValueError` to be raised? Should we be letting it get thrown so it gets recorded in Sentry, or are we okay with just silently suppressing it.

Reply:
I can't be sure because I don't know the entire context of all the files in the STAC collection or a guarantee of the similarity of the files.  If I allow errors to propagate up we then have a bunch of failed downloaded images.  I.E. that image could be at the beginning of a list of site downloads, breaking all images from downloading if you're downloading all sites.  I'm open to a way of recording this in sentry without causing the tasks to exit.  Most of this PR has been spent testing and finding new things about these images that completely break downloading images.  I'm sure more of these corner cases exist but I don't have the time to find all of them.  This allows the files that work to be downloaded while skipping the files don't without killing the entire task the first instance it hits a weird image in the list of 200 images.